### PR TITLE
adding app.py

### PR DESCRIPTION
**Pull request by shahab-github**

### CODE REVIEW

File: src/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Explaing how to use this script.
+### How to use this script
+This script is designed to be used with the following command line arguments:
+- `--input_file`: The input files should to be processed.
+- `--output_file`: The output file should  to be created.
+- `--num_lines`: The number of lines to be processed.
+- `--num_columns`: The number of columns must  be processed.
+- `--num_rows`: The number of rows must be processed.
+- `--num_files`: The number of files to be processed.
+
```
Comment (adevops01) [inline src/README.md:11]: Can you add more details about the files.

Comment (shahab-github) [reply src/README.md:11]: changed as suggest

File: app.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+print("Commit before creating a PR")
+
+# here are the changes that were added to this file as 
+# a part of new commit after creating the PR
+
+print("Changes after PR creation but before approval")
```
Comment (adevops01) [inline app.py]:
```suggestion
def printFuntion():
    print("Changes after PR creation but before approval")

```

Comment (adevops01) [reply app.py]: please review the chnages

File: src/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+###### Adding Readme file
+Explaing how to use this script.
+### How to use this script
```
Comment (adevops01) [inline src/README.md]:
```suggestion
### How to use this python script
```

File: app.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
+print("Commit before creating a PR")
+
+# here are the changes that were added to this file as 
+# a part of new commit after creating the PR
```
Comment (adevops01) [inline app.py:4]: typos, please correct typos

**Review by adevops01 — CHANGES_REQUESTED** [review]
please changes as suggest

**Review by adevops01 — APPROVED** [review]

**Pull request merged**